components/service/experiments/src/main/java/mozilla/components/service/experiments/ValuesProvider.kt

```diff
@@ -100,4 +101,8 @@ internal open class ValuesProvider {
     open fun getClientId(context: Context): String {
         return DeviceUuidFactory(context).uuid
     }
+
+    open fun getRandomBranchValue(lower: Int, upper: Int): Int {
```

Comment:
Can we document this function with `@param` and `@return` like the other functions in this class?